app/views/layouts/admin/_page.html.slim

```diff
@@ -23,8 +30,10 @@ div
 
           .division.centered-box.nav.navbar-nav.col-sm-6
             span> Division:
-            select.form-control
-              option All
+            / Todo: Sort out proper css display handling here
```

Comment:
This was a question that I raised to Slack yesterday.  I tried for a while, but I don't have the CSS skills to solve this.  Fuzzy pinged Elizabeth about it.